package.json

```diff
@@ -46,8 +48,8 @@
     "react-instantsearch-dom": "6.38.1",
     "react-parallax": "3.3.0",
     "react-redux": "7.2.2",
-    "react-router": "6.16.0",
-    "react-router-dom": "6.16.0",
+    "react-router": "6.18.0",
+    "react-router-dom": "6.18.0",
```

Comment:
[context] resolves a bug with `matchPath` and optional route parameters (e.g., `/:enterpriseSlug?`)